docs/assets/js/charts.js

```diff
@@ -1,703 +1,835 @@
+                    matrix[0][y] = t;
+                }
+                if (y == orgID)
+                {
+                    let t = matrix[x][y];
+                    matrix[x][y] = matrix[x][0];
+                    matrix[x][0] = t;
+                }
+            }
+        }
+    }
+
+    drawCoord(orgs, matrix);
 }
 
 function createCollaborationChart(canvas)
 {
-	const url = $(canvas).data("url");
-	const quota = 50;
-	let spinner = createSpinner(canvas);
-
-	d3.text(url,
-		function(text)
-		{
-			const data = d3.tsvParseRows(text);
-			const orgs = data.shift();
-			const matrix = data.map(function(x) { return x.map(function(y) { return +y; }) });
-
-			function menuChanged() {
-				// The rendering functions are going to adjust the org and
-				// matrix array. Therefore, we create a deep copy here.
-				const orgsCopy = orgs.slice(0);
-				const matrixCopy = matrix.map(function(x) { return x.slice(0); });
-
-				if (d3.event && +d3.event.target.value >= 0) {
-					visualizeSingleOrg(orgsCopy, matrixCopy, +d3.event.target.value);
-				} else {
-					visualizeOrgsWithTopConnections(orgsCopy, matrixCopy, quota);
-				}
-			}
-
-			const menuItems = [
-					{ value:-1, name:`Top ${quota} connections` },
-					{ value:-1, name:"—" },
-				].concat(
-				orgs.map(function(x, i) { return { value:i, name:x }; })
-			);
-			const select = d3.select("select")
-				.attr("class","select")
-				.on("change", menuChanged)
-				.selectAll("option")
-				.data(menuItems).enter()
-				.append("option")
-					.attr("value", function (d) { return d.value; })
-					.text(function (d) { return d.name; })
-
-			menuChanged();
-		}
-	).on("load.spinner", function() {
-		spinner.stop();
-	});
+    const url = $(canvas).data('url');
+    const quota = 50;
+    let spinner = createSpinner(canvas);
+
+    d3.text(url,
+        function(text)
+        {
+            const data = d3.tsvParseRows(text);
+            const orgs = data.shift();
+            const matrix = data.map(function(x)
+            {
+                return x.map(function(y)
+                {
+                    return +y;
+                });
+            });
+
+            function menuChanged()
+            {
+                // The rendering functions are going to adjust the org and
+                // matrix array. Therefore, we create a deep copy here.
+                const orgsCopy = orgs.slice(0);
+                const matrixCopy = matrix.map(function(x)
+                {
+                    return x.slice(0);
+                });
+
+                if (d3.event && +d3.event.target.value >= 0)
+                {
+                    visualizeSingleOrg(orgsCopy, matrixCopy, +d3.event.target.value);
+                }
+                else
+                {
+                    visualizeOrgsWithTopConnections(orgsCopy, matrixCopy, quota);
+                }
+            }
+
+            const menuItems = [
+                { value: -1, name: `Top ${quota} connections` },
```

Comment:
I wouldn’t put spaces after `{` and before `}` in general (also with inline function definitions).

Reply:
I've added the [`object-curly-spacing`](https://eslint.org/docs/rules/object-curly-spacing) rule to handle this, and also tweaked the [`brace-style` options](https://eslint.org/docs/rules/brace-style#options) to allow for inline function definition.